FmpDevicePkg/FmpDxe/FmpDxe.inf

```diff
@@ -76,6 +76,7 @@
   gFmpDevicePkgTokenSpaceGuid.PcdFmpDeviceTestKeySha256Digest              ## CONSUMES
   gFmpDevicePkgTokenSpaceGuid.PcdFmpDeviceImageTypeIdGuid                  ## CONSUMES
   gEfiMdeModulePkgTokenSpaceGuid.PcdTestKeyUsed                            ## SOMETIMES_PRODUCES
+  gFmpDevicePkgTokenSpaceGuid.PcdFmpDxeRequiredEKU                         ## CONSUMES  ## MU_CHANGE

```

Comment:
This commit needs to come before it gets used in SecurityPkg, right?

Reply:
yes! It does! don't know how they got out of order, will fix